test/normalCommand.test.ts

```diff
@@ -0,0 +1,74 @@
+import { newTest } from './testSimplifier';
+import { cleanUpWorkspace, setupWorkspace } from './testUtils';
+
+suite('Execute normal command', () => {
+  setup(async () => {
+    await setupWorkspace();
+  });
```

Comment:
```suggestion
  setup(setupWorkspace);
```

Reply:
Fix it.
[04500b0](https://github.com/VSCodeVim/Vim/pull/8896/commits/04500b01a9fb8c2e9f9b48b8c794c2e87857664d)